app/controllers/web-payments/apple-pay/merchant-validation.controller.js

```diff
@@ -66,92 +63,59 @@ module.exports = async (req, res) => {
-        res.status(200).send(response.data)
-      } catch (error) {
-        logger.info('Error generating Apple Pay session with axios and https proxy agent (hpagent)', {
-          ...getLoggingFields(req),
-          error: error.message,
-          status: error.response ? error.response.status : 'No status'
-        })
-        logger.info('Apple Pay session via axios and https proxy agent (hpagent) failed', 'Apple Pay Error')
-        res.status(500).send('Apple Pay Error')
-      }
-    } else {
-      logger.info('Generating Apple Pay session via axios and https proxy agent (hpagent) (NO PROXY)')
-      try {
-        const response = await axios(options)
+  if (proxyUrl) {
+    logger.info('Generating Apple Pay session via axios and https proxy agent (hpagent)')
 
-        logger.info('Apple Pay session successfully generated via axios and https proxy agent (hpagent) (NO PROXY)')
-        res.status(200).send(response.data)
-      } catch (error) {
-        logger.info('Error generating Apple Pay session (NO PROXY)', {
-          ...getLoggingFields(req),
-          error: error.message
-        })
-        logger.info('Apple Pay session via axios and https proxy agent (hpagent) with no proxy failed', 'Apple Pay Error')
-        res.status(500).send('Apple Pay Error')
-      }
+    const data = {
+      merchantIdentifier: merchantIdentityVars.merchantIdentifier,
+      displayName: 'GOV.UK Pay',
+      initiative: 'web',
+      initiativeContext: process.env.APPLE_PAY_MERCHANT_DOMAIN
     }
 
+    try {
+      const response = await axiosInstance.post(url, data, { headers: { 'Content-Type': 'application/json; charset=utf-8' } })
+
+      logger.info('Apple Pay session successfully generated via axios and https proxy agent (hpagent)')
+      res.status(200).send(response.data)
+    } catch (error) {
+      logger.info('Error generating Apple Pay session with axios and https proxy agent (hpagent)', {
+        ...getLoggingFields(req),
+        error: error.message,
+        status: error.response ? error.response.status : 'No status'
+      })
+      logger.info('Apple Pay session via axios and https proxy agent (hpagent) failed', 'Apple Pay Error')
+      res.status(500).send('Apple Pay Error')
+    }
   } else {
-    logger.info('Generating Apple Pay session via request retry')
-    request(options, (err, response, body) => {
-      if (err) {
-        logger.info('Error generating Apple Pay session', {
-          ...getLoggingFields(req),
-          error: err,
-          response: response,
-          body: body
-        })
-        logger.info('Apple Pay session via request retry failed', body)
-        return res.status(500).send(body)
-      }
-      logger.info('Apple Pay session successfully generated via request retry')
-      res.status(200).send(body)
-    })
+    logger.info('Generating Apple Pay session via axios and https proxy agent (hpagent) (NO PROXY)')
+    try {
+      const response = await axios(options)
```

Comment:
Is this path ever going to be hit? It shouldn't work as there's no `proxy: false` set in the options - axios will still try to use it's internal proxy and fail